research/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Build with the command
+# docker build -t webplotdigitizer:dev .
+
+# Run with the command to have port 8080 serve webplotdigitizer
+# docker run -p 8080:8080 webplotdigitizer:dev
+
+# Set the base image to a long-term Ubuntu release
+FROM ubuntu:18.04
+
+# Dockerfile Maintainer
+MAINTAINER William Denney <wdenney@humanpredictions.com>
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive \
+       apt-get install --yes --no-install-recommends \
+       git unzip wget ca-certificates python3 xz-utils libxml2 \
+       sudo tzdata \
+    && DEBIAN_FRONTEND=noninteractive \
+       ln -fs /usr/share/zoneinfo/UTC /etc/localtime \
+    && DEBIAN_FRONTEND=noninteractive \
+       dpkg-reconfigure --frontend noninteractive tzdata
+
+
+RUN git clone https://github.com/ankitrohatgi/WebPlotDigitizer.git \
+    && cd WebPlotDigitizer \
+    && sed s/wine// setupUbuntuDev.sh > setupUbuntuDev-nowine.sh \
```

Comment:
This is probably going to break now since I have changed `wine` to `wine-stable`.

Reply:
Good catch!